Framework/Kernel/src/Matrix.cpp

```diff
@@ -1613,7 +1613,7 @@ void fillFromStream(std::istream &is, Kernel::Matrix<T> &in,
 
 // Symbol definitions for common types
 template class MANTID_KERNEL_DLL Matrix<double>;
```

Comment:
I agree `MANTID_KERNEL_DLL` will be equivalent to `DLLExport` and should be changed. I am not sure though that we are consistent in general. We seem to have a mix of `DLL_Export` and library specific macros.